makefiles/S3.toml

```diff
@@ -58,8 +65,20 @@ script = "${HOME}/.fluvio/bin/fluvio install --develop fluvio/fluvio-package"
 
 [tasks.bump-fluvio]
 dependencies = ["install-fluvio-package", "bump-fluvio-latest"]
-script = """${HOME}/.fluvio/bin/fluvio package --test bump dynamic ${FLUVIO_VERSION} """
+script = """
+${HOME}/.fluvio/extensions/fluvio-package \
+    ${FLUVIO_PUBLISH_TEST} \
```

Comment:
why it is using `FLUVIO_PUBLISH_TEST` ?

Reply:
`FLUVIO_PUBLISH_TEST` has two possible values: empty string `""` or `--test`. When it it `--test`, this will attempt to run the  bump command against `/test/` in the package registry instead of `/v1/`, so we can verify the effect of the command.

Reply:
Does this work? I tried to do this sort of thing with the github-release command. I think the "" case might actually pass in the commandline parameter of "" which wasn't acceptable for `github-release`. It might be okay with fluvio-publish.

Reply:
Yes, this works if you put it in a `script` block. If you wrote it as a task with `command` and `args` it would not work, you're right.